info/v2/conversion.go

```diff
@@ -96,7 +96,7 @@ func MachineStatsFromV1(cont *v1.ContainerInfo) []MachineStats {
 	return stats
 }
 
-func ContainerStatsFromV1(containerName string, spec *v1.ContainerSpec, stats []*v1.ContainerStats) []*ContainerStats {
+func ContainerStatsFromV1(containerName string, spec *v1.ContainerSpec, stats []*v1.ContainerStats, firstCall bool) []*ContainerStats {
```

Comment:
or an alternative is to update this to return `[]*ContainerStats, []warnings`, and then we can optionally log warnings or not from the caller?